build.gradle

```diff
@@ -441,8 +441,9 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
   integrationTestImplementation group: 'net.minidev', name: 'json-smart', version: '2.4.7'
-
   integrationTestImplementation group: 'org.testcontainers', name: 'postgresql', version: versions.testcontainers
+  integrationTestImplementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
```

Comment:
Is importing lombok dependency just to be able to use `val` keyword worth it ?

It also makes code read bit inconsistent ?

Reply:
Lombok is nice, val is good . :) hehe

Reply:
I don't like it, but again guess it's down to personal preference